### PR TITLE
add some runtime related faq

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

docs update

* **What is the current behavior?** (You can also link to an open issue here)

Currently, there are many community issues related to the runtime, but we do not have corresponding FAQs for them.

* **What is the new behavior (if this is a feature change)?**

Add FAQs specifically related to the runtime to assist users in troubleshooting and resolving issues.

### CODE REVIEW

File: docs/faq/setup.md [hunk 1]
```diff
@@ -381,4 +381,108 @@ F1121 15:21:15.154526 3671032 certmanager.go:94] Error: failed to get CA certifi
 
 One more important thing about `cloudcore` container mode is about how to expose cloudcore port to edge nodes. In container mode, we will also create a cloudcore service. And it's your duty to choose a LoadBalancer or adjust it to `NodePort` ServiceType, to expose `cloudcore` service to edge nodes. For more details, please reference [k8s service docs](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)
 
+## Runtime related issues
```
Comment (fisherxu) [inline docs/faq/setup.md:384]: How about moving the `Runtime related issues` to the top? As there're many issues releted to them.

Comment (Shelley-BaoYue) [reply docs/faq/setup.md:384]: > How about moving the `Runtime related issues` to the top? As there're many issues releted to them.

 I find the preceding content is also significant, so I placed it here. Users can use the directory on the right to quickly locate it.   But this directory can be further organized for better clarity and ease of use.

image


Alternatively, we can add subdirectories below the setup, resulting in the following effect. 

image

Comment (fisherxu) [reply docs/faq/setup.md:384]: subdirectories looks great.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**